### ISSUE TITLE
Homebridge error

### ISSUE THREAD

**Issue by Joeip12** [nfarina/homebridge#1342]
Hi Guys, 

I'm a newbie for coding, so need some help to resolve the below problem. 

I recently setup a homebridge with my tp-link smart plug (HS100). It run ok in first few days.  but i found there are some error these days, could anyone please give me some to solve the problem.  Thanks in advance.

 [6/12/2017, 9:53:23 PM] Loaded plugin: homebridge-hs100
[6/12/2017, 9:53:23 PM] Registering platform 'homebridge-hs100.Hs100'
[6/12/2017, 9:53:24 PM] ---
[6/12/2017, 9:53:24 PM] Loaded config.json with 0 accessories and 1 platforms.
[6/12/2017, 9:53:24 PM] ---
[6/12/2017, 9:53:24 PM] Loading 1 platforms...
[6/12/2017, 9:53:24 PM] Initializing Hs100 platform...
[6/12/2017, 9:53:24 PM] Loading 0 accessories...
Scan this code with your HomeKit App on your iOS device to pair with Homebridge:
                       
    ┌────────────┐     
    │ 031-45-154 │     
    └────────────┘     
                       
events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE :::51826
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at Server._listen2 (net.js:1236:14)
    at listen (net.js:1272:10)
    at Server.listen (net.js:1368:5)
    at EventedHTTPServer.listen (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/lib/util/eventedhttp.js:60:19)
    at HAPServer.listen (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/lib/HAPServer.js:158:20)
    at Bridge.Accessory.publish (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/lib/Accessory.js:509:16)
    at Server._publish (/usr/local/lib/node_modules/homebridge/lib/server.js:114:16)
    at Server.run (/usr/local/lib/node_modules/homebridge/lib/server.js:91:10)

**Comment by snowdd1** [comment]
Might be a duplicate of https://github.com/nfarina/homebridge/issues/390#issuecomment-161108134 ?

**Comment by EdwinJS** [comment]
Yeah a duplicate, run sudo killall homebridge before starting homebridge, it may be a case of another homebridge instance running already
